Metro-Records/la-metro-councilmatic
# Migrate alert management to Wagtail

## Overview

See title. Connects #1175.

### Demo

<img width="398" alt="Screenshot 2024-11-22 at 9 58 46 AM" src="https://github.com/user-attachments/assets/34205ef2-9b3c-432a-afb5-a523d9908f09">
<img width="1512" alt="Screenshot 2024-12-04 at 1 34 07 PM" src="https://github.com/user-attachments/assets/c02c0b3e-0d20-44af-88c3-44a0190d0029">
<img width="1512" alt="Screenshot 2024-12-04 at 1 34 14 PM" src="https://github.com/user-attachments/assets/1d277308-5b4a-4da0-af8f-f0f4961a3ba2">
<img width="1512" alt="Screenshot 2024-12-04 at 1 34 59 PM" src="https://github.com/user-attachments/assets/38bd4059-87e8-463a-aa5e-c6e3f712d66f">

## Testing Instructions

_Credentials for the datamade user are in Bitwarden under `LA Metro Wagtail User - Staging`_

- Using the review app, add some alerts with various rich text elements. Confirm they appear on the site as expected.
- Also confirm that there's a link to manage alerts in the Wagtail userbar in the bottom left.
